### PR TITLE
Vertex color tileset

### PR DESCRIPTION
**Pull request by lilleyse**
Generates a tileset that uses vertex colors. CC https://github.com/AnalyticalGraphicsInc/cesium/pull/6089

### CODE REVIEW

**Comment by pjcozzi** [comment]
@emackey any chance you can look at this?

**Comment by emackey** [comment]
I haven't had a chance to dig into this repo and its uses yet, but the code looks OK at a glance.

**Comment by pjcozzi** [comment]
Thanks @emackey.  @shehzan10 could you take a closer look?

File: samples-generator/lib/Mesh.js [hunk 1]
```diff
@@ -130,6 +136,23 @@ Mesh.prototype.setPositionsRelativeToCenter = function() {
...
+/**
+ * Bake materials as vertex colors. Use the default white opaque material.
+ */
+Mesh.prototype.transferMaterialToVertexColors = function() {
+    var material = this.material;
+    this.material = whiteOpaqueMaterial;
+    var vertexCount = this.getVertexCount();
+    var vertexColors = new Array(vertexCount * 4);
+    this.vertexColors = vertexColors;
+    for (var i = 0; i < vertexCount; ++i) {
```
Comment (shehzan10) [inline samples-generator/lib/Mesh.js]: What do you think about using different per-vertex colors to give it a "colorful" feel? I feel like that would make a better sample.

Comment (lilleyse) [reply samples-generator/lib/Mesh.js]: That would demonstrate vertex colors better, but I think this is alright for now. It's also easier to work with for the color blend mode test in Cesium which requires the center building to be red.

**Pull request merged**